### ISSUE TITLE
ios 9 shelf issue after rotate

### ISSUE THREAD

**Issue by quba88** [bakerframework/baker#506]
When rotate device with iOS 9, screen is half black.

![73b998b555a2c00c0e21f26096cc212cfba423882fff7385dfpimgpsh_fullsize_distr](https://cloud.githubusercontent.com/assets/1645706/10041909/4cebae24-61e8-11e5-8b1a-9dd7f6508055.jpg)

**Comment by MagLoftMagazines** [comment]
Hi @quba88, sorry to hear about your troubles. Please describe in a little more detail what is happening and when. Is it on all device types (iPhones, iPads), what build version does it happen on, etc?

What version of Baker are you using?

Have you had a look by searching for earlier reports using "rotate":
https://github.com/bakerframework/baker/issues?utf8=%E2%9C%93&q=rotate

**Comment by quba88** [comment]
Hi @nicholasmartin , I found problem and workaround. The problem is in the method:
- (void)willAnimateRotationToInterfaceOrientation:(UIInterfaceOrientation)toInterfaceOrientation duration:(NSTimeInterval)duration, I removed this method and now work fine.

Also is problem with Table of Contents. In ios9 is too smal and to high in view. Workaround replace the meta data:
these
<meta name='viewport' content='width=13530, height=197, initial-scale=1.0, maximum-scale=1.0, user-scalable=no' />
to
<meta name='viewport' content='width=13530, height=197, initial-scale=2.0, maximum-scale=2.0, user-scalable=no' />

and, lock scrollView in verticaly
